config/plugins/visualizations/unipept/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
```

Comment:
@caleb-easterly are you also fine to distribute this under the same license than the Galaxy project? If so you could remove this file as well.